modules/client/src/addressList/internal/client/methods.ts

```diff
@@ -104,29 +105,29 @@ export class ClientAddressListMethods extends ClientCore
     };
 
     const receipt = await tx.wait();
-    const addresslistContractInterface = AllowlistVoting__factory
+    const addresslistContractInterface = AddresslistVoting__factory
       .createInterface();
-    const log = receipt.logs.find(
-      (log) =>
-        log.topics[0] ===
-          id(
-            addresslistContractInterface.getEvent("VoteCreated").format(
-              "sighash",
-            ),
-          ),
+
+    const log = findLog(
```

Comment:
Shouldn't it be called `findEvent` maybe? not important

Reply:
Well, the function returnrns a `Log`